src/js/helpers/error-handler.js

```diff
@@ -2,17 +2,29 @@ import { errors } from '@/js/errors'
 import { Bus } from '@/js/helpers/event-bus'
 import log from 'loglevel'
 import i18next from 'i18next'
+import { ErrorTracker } from '@/js/helpers/error-tracker'
 
 export class ErrorHandler {
-  static process (error, translationId = '') {
-    ErrorHandler.processWithoutFeedback(error)
-    Bus.error(translationId || ErrorHandler._getTranslationId(error))
+  static process (error, translationId = '', sentryReportConfig = {}) {
```

Comment:
since there are no mentions of sentry anymore, you call `ErrorTracker.trackMessage()`, you cannot call the tracker config with a `sentryReportConfig` anymore. Rename the former sentry-related variable and messages appropriately.

Reply:
Renamed

Reply:
renamed